testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -6680,3 +6688,1393 @@ fn test_flash_block_skip_tenure() {
 
     channel.stop_chains_coordinator();
 }
+
+#[test]
+#[ignore]
+fn exit_at_rc_integration_test() {
```

Comment:
is this really one indivisiable test?

Reply:
There are probably a few I could split out, but a lot of the test cases within are dependent on each other. 

Also, the advantage of testing these in one integration test is that we can see how a variety of different scenarios would inter-play with each other (giving us greater confidence for this contract being deployed on our chain)